docker/standalone.xml

```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0" ?>
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I found this change in the generated `distribution/server-dist/target/keycloak-3.2.0.Final/standalone/configuration/standalone.xml`